spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'foods', type: :feature do
+  describe 'I can input food query to view returned results' do
+    it "I can input 'sweet potatoes and can see the associated information " do 
+      visit root_path
+
+      fill_in :q, with: "sweet potatoes"
+      click_button "Search"
+
+      expect(current_path).to eq(foods_path)
+
+
+      expect(page).to have_content("Searched item count: 44128")
+
+      expect(page).to have_css('.food_information', count: 10)
```

Comment:
Excellent tests that follow the spec exactly. However the actual count of the `Searched Item Count` could break anytime the API changes. It would have been best to stub this request with data you can depend on remaining consistent.